### PR TITLE
Refactoring FragmentCache due yiisoft/view changes

### PR DESCRIPTION
**Pull request by devanych**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | -

Waiting: https://github.com/yiisoft/view/pull/159

### CODE REVIEW

**Review by terabytesoftw — APPROVED** [review]

File: src/FragmentCache.php [hunk 1]
```diff
@@ -4,48 +4,48 @@
...
-final class FragmentCache extends Widget implements DynamicContentAwareInterface
-{
-    use DynamicContentAwareTrait;
+use function ob_end_clean;
+use function ob_get_clean;
+use function ob_implicit_flush;
+use function ob_start;
 
-    private string $id;
+final class FragmentCache extends Widget
```
Comment (samdark) [inline src/FragmentCache.php:19]: It would be great to have phpdoc for it with a usage example.

File: src/FragmentCache.php [hunk 2]
```diff
@@ -4,48 +4,48 @@
...
-    private int $duration = 60;
     private ?Dependency $dependency = null;
-    private $variations;
-    private ?string $content = null;
+    private int $ttl = 60;
+    private array $variations = [];
 
-    private WebView $webView;
+    /**
+     * @var array<string, DynamicContent>
```
Comment (samdark) [inline src/FragmentCache.php:28]:
```suggestion
     * @var DynamicContent[]
```

Comment (devanych) [reply src/FragmentCache.php:28]: No. Then you can add different objects with the same `ID`.

File: src/FragmentCache.php (hunk 1 above)
Comment (devanych) [reply src/FragmentCache.php:39]: Done.

**Pull request merged**